src/sentry/incidents/action_handlers.py

```diff
@@ -241,6 +241,21 @@ def send_alert(
             self.record_alert_sent_analytics(self.action.target_identifier, notification_uuid)
 
 
+@AlertRuleTriggerAction.register_type(
+    "discord",
+    AlertRuleTriggerAction.Type.DISCORD,
+    [AlertRuleTriggerAction.TargetType.SPECIFIC],
+    integration_provider="discord",
+)
+class DiscordActionHandler(DefaultActionHandler):
+    def send_alert(self, metric_value: int | float, new_status: IncidentStatus):
+        from sentry.integrations.discord.actions.metric_alert import (
+            send_incident_alert_notification,
+        )
+
+        send_incident_alert_notification(self.action, self.incident, metric_value, new_status)
```

Comment:
please emit the analytic event that the others are doing

```
  if success:
      self.record_alert_sent_analytics(self.action.target_identifier, notification_uuid)
```

Reply:
this will also require setting a provider see `provider = "pagerduty"` like the others